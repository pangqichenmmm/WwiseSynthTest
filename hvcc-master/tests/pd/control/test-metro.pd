#N canvas 494 23 683 554 10;
#X obj 167 63 loadbang;
#X obj 270 193 + 1;
#X obj 167 85 t b b;
#X msg 221 129 1;
#X obj 235 159 metro 50;
#X obj 221 332 * 10;
#X obj 167 382 metro 2;
#X obj 167 474 print;
#X msg 85 262 0;
#X obj 235 193 f 0;
#X msg 217 257 10 \$1;
#X obj 217 279 -;
#X obj 167 429 f;
#X obj 221 354 t f f;
#X obj 217 301 t f f;
#X obj 85 284 t f f;
#X obj 84 237 sel 0;
#X connect 0 0 2 0;
#X connect 1 0 9 1;
#X connect 2 0 6 0;
#X connect 2 1 3 0;
#X connect 3 0 4 0;
#X connect 4 0 9 0;
#X connect 5 0 13 0;
#X connect 6 0 12 0;
#X connect 8 0 15 0;
#X connect 9 0 1 0;
#X connect 9 0 10 0;
#X connect 10 0 11 0;
#X connect 11 0 14 0;
#X connect 12 0 7 0;
#X connect 13 0 6 1;
#X connect 13 1 12 1;
#X connect 14 0 5 0;
#X connect 14 1 16 0;
#X connect 15 0 6 0;
#X connect 15 1 4 0;
#X connect 16 0 8 0;
