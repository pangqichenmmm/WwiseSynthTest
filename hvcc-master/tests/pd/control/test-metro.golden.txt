[@ 0.000] print: 100
[@ 100.000] print: 90
[@ 190.000] print: 70
[@ 260.000] print: 50
[@ 310.000] print: 40
[@ 350.000] print: 30
[@ 380.000] print: 30
[@ 410.000] print: 20
[@ 430.000] print: 20
[@ 450.000] print: 10
[@ 460.000] print: 10
[@ 470.000] print: 10
[@ 480.000] print: 10
[@ 490.000] print: 10
